tests/Mindee.UnitTests/Extraction/ImageExtractorTest.cs

```diff
@@ -0,0 +1,128 @@
+using Mindee.Extraction;
+using Mindee.Input;
+using Mindee.Parsing.Common;
+using Mindee.Product.BarcodeReader;
+using Mindee.Product.MultiReceiptsDetector;
+
+namespace Mindee.UnitTests.Extraction
+{
+    [Trait("Category", "ImageExtractor")]
+    public class ImageExtractorTest
+    {
+        [Fact]
+        public async Task GivenAnImage_ShouldExtractPositionFields()
+        {
+            var image = new LocalInputSource("Resources/products/multi_receipts_detector/default_sample.jpg");
+            var response = await GetMultiReceiptsDetectorPrediction("complete");
+            var inference = response?.Document.Inference;
+
+            var extractor = new ImageExtractor(image);
+            Assert.Equal(1, extractor.GetPageCount());
+
+
+            if (inference != null)
```

Comment:
should never be null

Reply:
IDE warning. I can remove it but I don't think it hurts, especially inside a unit test

Reply:
all the other unit tests don't have this and are fine